pytest_fixtures/component/leapp_client.py

```diff
@@ -163,7 +163,9 @@ def custom_leapp_host(upgrade_path, module_target_sat, module_sca_manifest_org,
         host_class=ContentHost,
         deploy_rhel_version=upgrade_path['source_version'],
         deploy_flavor=settings.flavors.default,
-        deploy_network_type='ipv6' if settings.server.is_ipv6 else 'ipv4',
+        # TODO(shwsingh): Check whether this is valid for dualstack scenaro. Ideally, we should
+        # parametrize this fixture by fixture_markers plugin
+        deploy_network_type=settings.content_host.attributes.network_type,
```

Comment:
We are also not supporting Leapp for dualstack, so we can skip this

Reply:
Removing the TODO comment